debian/rules

```diff
@@ -5,9 +5,12 @@
 
 export DH_VERBOSE=1
 
-export DEB_BUILD_MAINT_OPTIONS = hardening=+all
-DPKG_EXPORT_BUILDFLAGS = 1
-include /usr/share/dpkg/buildflags.mk
+## Do not enable hardening compile flags per:
+## https://github.com/ResistancePlatform/resistance-miner/issues/8#issuecomment-519093678
+## https://github.com/ResistancePlatform/resistance-miner/issues/9
+#export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+#DPKG_EXPORT_BUILDFLAGS = 1
+#include /usr/share/dpkg/buildflags.mk
```

Comment:
With this commented out, what build flags are actually used?

Reply:
That's the problem. Already created https://github.com/ResistancePlatform/resistance-miner/issues/12 for it for this to be discussed separately.